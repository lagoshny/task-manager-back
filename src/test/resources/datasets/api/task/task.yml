TASK:
  - ID: 1
    NUMBER: 1
    NAME: "Test task"
    PRIORITY: 1
    STATUS: 0
    CREATION_DATE: "2019-02-06 18:19:13.176000"
    NEED_TIME_MANAGEMENT: 0
    CATEGORY_ID: 1
    TOTAL_TIME: 30
    AUTO_REDUCE: 0
    AUTHOR_ID: 1

TASK_CATEGORY:
  - ID: 1
    DESCRIPTION: "Default category"
    ICON: "fa-certificate"
    NAME: "Default"
    PREFIX: "default"
    USER_ID: 1