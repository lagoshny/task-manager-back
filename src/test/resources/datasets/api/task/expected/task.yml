TASK:
  - NUMBER: 1
    NAME: "Name"
    PRIORITY: 2
    STATUS: 0
    NEED_TIME_MANAGEMENT: 1
    TOTAL_TIME: 10
    AUTO_REDUCE: 1
    AUTHOR_ID: 1